plugins/source/github/client/alert.json

```diff
@@ -0,0 +1,1195 @@
+{
```

Comment:
Is this file still being used anywhere? I didn't find it using cmd+f on the PR, but I may have missed it?

Reply:
didn't look for it, will check

Reply:
removed extra file